std/utf.d

```diff
@@ -3110,7 +3110,8 @@ enum dchar replacementDchar = '\uFFFD';
  * Params:
  *      r = input range of characters, or array of characters
  * Returns:
- *      input range
+ *      An input range with a length if `isAggregateType!R` is `true`.
+ *      Otherwise, this returns a random access range with slicing.
```

Comment:
Maybe state that it has length in the second case, too. That doesn't strictly follow from random access range with slicing.